src/pages/aurora-site/learning/speaking-content.jsx

```
@@ -1,83 +1,66 @@
-import {
-  Card,
-  CardContent,
-  CardHeader,
-  CardTitle,
-} from "@/components/layout/ui/card";
-import { Progress } from "@/components/layout/ui/progress";
-import { CheckCircle, Lock, Mic } from "lucide-react";
+import React from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)